src/screens/ContactDetail/components/ContactHeader.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Text, Pressable, ImageBackground } from 'react-native';
+
+import { Icon } from '../../../components/Icon';
+import { colors } from '../../../util/colors';
+import images from '../../../assets/images';
+import { NavIconButton } from '../../../components/NavIconButton';
+import Avatar from '../../../components/Avatar/Avatar';
+import styles from './styles';
+
+type ContactHeaderProps = {
+  navigation: any;
+  isEditing?: boolean;
+  onPressEdit?: () => void;
+  avatar?: string;
+  email?: string;
+  name?: string;
+};
+
+export const ContactHeader = ({
+  navigation,
+  isEditing,
+  avatar,
+  email,
+  name,
+  onPressEdit = () => {},
```

Comment:
why do we need default function?

Reply:
I added this because I wanted to use like this: onPress={onPressEdit}.
But okay I'll delete

Reply:
Why you can use onPress={onPressEdit}

Reply:
I changed this.